dev/io.openliberty.microprofile.openapi.4.1.internal_fat_tck/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src
+
+javac.source: 11
+javac.target: 11
+
+fat.project: true
+
+tested.features: \
+  validation-3.1,\
+  expressionlanguage-5.0,\
+  expressionlanguage-6.0,\
+  restfulWS-3.0,\
+  restfulWS-4.0,\
+  servlet-6.0,\
+  servlet-7.0
```

Comment:
```suggestion
  restfulWS-4.0
```
I don't believe you need the other ones.  restfulWS-3.0 is EE 9 and definitely wouldn't be used and servlet-7.0 isn't a thing.  For EE 11 it is servlet-6.1, but I don't think you use servlet for these tests.

Reply:
I think you're right, this list looks like it hasn't been updated.

Reply:
Test complains if el-5.0 is removed so I put it back. Not sure why, I assume there's some tolerates going on.